shared/js/otSdkDriver.js

```diff
@@ -776,6 +789,21 @@ loop.OTSdkDriver = (function() {
     },
 
     /**
+     * Sends the cursor position on the data channel.
+     *
+     * @param {String} message The message to send.
+     */
+    sendCursorMessage: function(message) {
+      if (!this._publisherChannel || !this._subscriberChannel) {
+        return;
```

Comment:
I was going to suggest checking that the publisher channel was subscribed to at the other end, unfortunately the SDK doesn't give us events to know that. So I think checking for publisher & subscriber makes sense. A comment in the code would be useful.